### PR TITLE
refactor along content

### PR DESCRIPTION
**Pull request by MariePMcGarry**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/metrics/introduction/joins.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+Query #B returns three result sets:
+
+```
+metric=CPU_Sys, account=dev
+metric=CPU_Sys, account=stage
+metric=CPU_Sys, account=prod
+```
+
+Query #C returns 9 result sets
```
Comment (stacykor) [inline docs/metrics/introduction/joins.md:78]:
```suggestion
Query #C returns 9 result sets:
```

File: docs/metrics/introduction/joins.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+#A: metric=CPU_User _source=”HostMetrics”
+#B: metric=CPU_User _source=”JBM Host Metrics”
+#C: #A+#B/2
+```
+
+Doing a quick comparison of metrics and average value over a short period can yield can surprising results, as the screenshot below illustrates.
+
+<img src={useBaseUrl('img/metrics/odd-join-results.png')} alt="odd-join-results.png"/>
+
+Note that query #C computed the two values of the average value.
```
Comment (stacykor) [inline docs/metrics/introduction/joins.md:129]:
```suggestion
:::note
 query #C computed the two values of the average value.
::: 
```

File: docs/metrics/introduction/joins.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+```sql
+#A: metric=bytes_in node=* cluster=*
+#B: metric=bytes_out node=* cluster=*
+#C: #A + #B along cluster, node as bytes_total
+```
+
+### Calculate percentages for nodes by cluster
+
+This query calculates `bytes_in` as a percentage of `bytes_total` for each node. The expression allows referring to the same series more than once in the expression: #A appears twice in the expression.
```
Comment (stacykor) [inline docs/metrics/introduction/joins.md:175]:
```suggestion
This query calculates `bytes_in` as a percentage of `bytes_total` for each node. The expression allows referring to the same series more than once in the expression, in this example #A appears twice in the expression.
```

**Review by stacykor — DISMISSED** [review]

**Review by kimsauce — APPROVED** [review]

**Review by kimsauce — APPROVED** [review]

File: docs/metrics/introduction/joins.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+#B: metric=CPU_User _source=”JBM Host Metrics”
+#C: #A+#B/2
+```
+
+Doing a quick comparison of metrics and average value over a short period can yield can surprising results, as the screenshot below illustrates.
+
+<img src={useBaseUrl('img/metrics/odd-join-results.png')} alt="odd-join-results.png"/>
+
+:::note
+ query #C computed the two values of the average value.
```
Comment (kimsauce) [inline docs/metrics/introduction/joins.md:130]:
```suggestion
query `#C` computed the two values of the average value.
```

File: docs/metrics/introduction/joins.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+
+```sql
+#A: metric=bytes_in node=* cluster=*
+#B: metric=bytes_out node=* cluster=*
+#C: #A + #B along cluster, node as bytes_total
+```
+
+### Calculate percentages for nodes by cluster
+
+This query calculates `bytes_in` as a percentage of `bytes_total` for each node. The expression allows referring to the same series more than once in the expression, in this example #A appears twice in the expression.
```
Comment (kimsauce) [inline docs/metrics/introduction/joins.md:177]:
```suggestion
This query calculates `bytes_in` as a percentage of `bytes_total` for each node. The expression allows referring to the same series more than once in the expression. In this example, `#A` appears twice in the expression.
```

File: docs/metrics/introduction/joins.md (hunk 4 above)
Comment (MariePMcGarry) [reply docs/metrics/introduction/joins.md]: @kimsauce, I format #C as code in query examples, like this:

#C: #A + #B along account

But not when referring to it as a label. So, let's not change it.

File: docs/metrics/introduction/joins.md (hunk 5 above)
Comment (MariePMcGarry) [reply docs/metrics/introduction/joins.md]: As above.

**Pull request merged**